src/extension/wps/src/main/java/au/org/emii/gogoduck/download/Config.java

```diff
@@ -0,0 +1,62 @@
+package au.org.emii.gogoduck.download;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * Configuration setting for the ParallelDownloadManager
+ */
+public class Config {
```

Comment:
We have a class with similar name au.org.emii.core.Config. Maybe we can rename it to something like DownloadConfig